Doc/library/stdtypes.rst

```diff
@@ -4575,7 +4575,7 @@ can be used interchangeably to index the same dictionary entry.
       such as an empty list.  To get distinct values, use a :ref:`dict
       comprehension <dict>` instead.
 
-   .. method:: get(key[, default])
+   .. method:: get(key[, default=None])
```

Comment:
```suggestion
   .. method:: get(key, default=None)
```

FYI: this would be more controversial (don't apply it! 😄):

```suggestion
   .. method:: get(key, default=None, /)
```